packages/firestore/src/remote/watch_change.ts

```diff
@@ -409,16 +411,81 @@ export class WatchChangeAggregator {
         }
       } else {
         const currentSize = this.getCurrentDocumentCountForTarget(targetId);
+        // Existence filter mismatch. Remove documents to limbo, and raise a
```

Comment:
"Remove documents to limbo" is a bit of an usual term. Could you say "Mark the documents as being in limbo" or something like that?